modules/apps/asset/asset-browser-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -136,7 +136,7 @@
 						<c:when test='<%= Objects.equals(assetBrowserDisplayContext.getDisplayStyle(), "icon") %>'>
 
 							<%
-							row.setCssClass("entry-card lfr-asset-item");
+							row.setCssClass("card-page-item card-page-item-asset");
```

Comment:
Are these classes always expected in a card-view?

Reply:
Question is... could we add them by default [here](https://github.com/liferay/liferay-portal/blob/master/portal-web/docroot/html/taglib/ui/search_iterator/lexicon/icon.jsp#L98) so we don't have to add them manually elsewhere?

Reply:
Yes we could. We could probably do something similar to https://github.com/liferay/liferay-portal/blob/master/portal-web/docroot/html/taglib/ui/search_iterator/lexicon/icon.jsp#L22-L24 where we output by default unless there are classes passed in?